cmd/podman/play/kube.go

```diff
@@ -23,6 +24,7 @@ type playKubeOptionsWrapper struct {
 	TLSVerifyCLI   bool
 	CredentialsCLI string
 	StartCLI       bool
+	StaticIPCLI    string
```

Comment:
Not needed see other comment

Reply:
flags will make things much easier, will update that